### DISCUSSION TITLE
useModel 不能在 app.tsx 文件中使用

### DISCUSSION THREAD

**Discussion by crazyair** [umijs/umi#12025] [Q&A]
umi3 一直是可以的，为什么 umi4 就不可以了，还报错 `TypeError: Cannot read properties of null (reading 'dispatcher')`
不想改数据流方案，有什么好的办法解决这个问题，复现链接 https://github.com/crazyair/antd5/tree/umi-use-model

**Comment by fz6m** [comment]
See https://github.com/umijs/umi/issues/10334#issuecomment-1878089397
